setup.py

```diff
@@ -1,13 +1,21 @@
-import setuptools
+from setuptools import find_packages, setup
+import os
 
 version = '2.9.1'
 
-setuptools.setup(
+try:
+    with open(os.path.join(os.path.dirname(__file__), 'README.md'), encoding='utf-8') as f:
```

Comment:
Is this sort of thing typical on PyPi?

Reply:
i found it in the Django ```setup.py``` [here](https://github.com/django/django/blob/782d85b6dfa191e67c0f1d572641d8236c79174c/setup.py#L64)